packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -120,43 +183,174 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
 
   return (
     <Box
-      className={cx(
-        'iui-expandable-block',
-        {
-          'iui-with-caption': !!caption,
-          'iui-expanded': expanded,
-          'iui-small': size === 'small',
-          'iui-borderless': styleType === 'borderless',
-        },
-        className,
+      className={cx('iui-expandable-header', className)}
+      role='button'
+      aria-expanded={isExpanded ?? !disabled}
+      aria-disabled={disabled}
+      tabIndex={0}
+      onClick={handleToggle}
+      onKeyDown={onKeyDown}
+      ref={forwardedRef}
+      {...rest}
+    >
+      {children ?? (
+        <>
+          {expandIcon && <ExpandableBlock.ExpandIcon />}
+          <ExpandableBlock.LabelArea>
+            <ExpandableBlock.Title>{label}</ExpandableBlock.Title>
+          </ExpandableBlock.LabelArea>
+        </>
       )}
-      style={style}
-      ref={ref}
+    </Box>
+  );
+}) as PolymorphicForwardRefComponent<'div', ExpandableBlockHeaderOwnProps>;
+ExpandableBlockHeader.displayName = 'ExpandableBlock.Header';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.ExpandIcon component
+
+const ExpandableBlockExpandIcon = React.forwardRef((props, forwardedRef) => {
+  const { className, children, ...rest } = props;
+  return (
+    <Box
+      className={cx('iui-expandable-block-icon', className)}
+      ref={forwardedRef}
       {...rest}
     >
+      {children ?? <SvgChevronRight aria-hidden />}
+    </Box>
+  );
+}) as PolymorphicForwardRefComponent<'div'>;
+ExpandableBlockExpandIcon.displayName = 'ExpandableBlock.ExpandIcon';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.LabelArea component
+
+const ExpandableBlockLabelArea = polymorphic.span('iui-expandable-block-label');
+ExpandableBlockLabelArea.displayName = 'ExpandableBlock.LabelArea';
+
+// ----------------------------------------------------------------------------
+// ExpandableBlock.Title component
+
+const ExpandableBlockTitle = React.forwardRef((props, forwardedRef) => {
+  const { className, children, ...rest } = props;
+  return (
+    <Box className={cx('iui-title', className)} ref={forwardedRef} {...rest}>
```

Comment:
Maybe it's possible to change classNames to be unique? Eg. `iui-expandable-block-title`

Reply:
Updated!